css/main-styling.css

```diff
@@ -1,89 +1,152 @@
-body {
-  font-family: 'Roboto Slab', serif;
-  font-size: 18px;
-  line-height: 1.5;
-  cursor: default;
-  font-color: black;
+.clearfix:after {
+  content: "";
+  display: table;
+  clear: both;
 }
 
-.wrapper {
-  max-width: 1280px;
-  margin: auto;
+html {
+  font-family: 'Roboto', serif;
 }
 
-h1 {
-  font-size: 2em;
-  line-height: 1;
+header {
+  background-color: #97c0e6;
 }
 
 h3 {
-  font-size: 1.25em;
+  font-family: 'roboto slab', serif;
+  font-size: 1.5em;
   font-weight: bold;
+  margin: 10px 0;
 }
 
-p {
-  margin: 0 0 1.5em 0;
-  font-size: 14px;
+.logo {
+  float: left;
 }
 
-.button {
-  text-decoration: none;
-  text-transform: uppercase;
-  letter-spacing: 1px;
-  font-weight: bold;
-  color: inherit;
-  display: block;
-  text-align: center;
-  box-shadow: inset 0 -3px 0 rgba(0,0,0,.2);
-  border-radius: 5px;
+.logo img {
+  padding: 15px 30px;
 }
 
-#page-header {
-  background: #97c0e6;
-  overflow: hidden;
-  padding: 0 3%;
+nav {
+  float: right;
+  font-family: 'roboto slab', serif;
+  color: white;
+  font-size: 1.5em;
+  font-weight: bold;
+  margin-right: 30px;
 }
 
-/*#page-header h1 {
-  insert img logo
-}*/
+nav li {
+  display: inline-block;
+  padding: 50px 20px;
+  line-height: 5px;
+}
 
-#page-header a {
-  color: white;
+a {
   text-decoration: none;
+  color: inherit;
 }
 
-#page-header nav {
-  float: right;
+a:hover {
+  color: black;
 }
 
-#page-header nav li {
-  float: left;
+.banner {
+  padding: 0 20px;
+  position: relative;
+  top: inherit;
 }
 
-#page-header nav a {
+.banner img {
   display: block;
-  /*double check display after logo is input*/
-  padding: 20px 15px;
+  margin: 15px auto;
 }
 
-#page-header nav li a:hover {
-  color: inherit;
-  font-weight: bold;
+.wrapper {
+  display: block;
+  max-width: 950px;
+  margin: 30px auto;
+  padding: 0 20px;
 }
 
-#page-header nav strong {
-  text-transform: uppercase;
+.action1-4 {
+  float: left;
+  width: 22%;
+  text-align: center;
+  font-size: 1em;
+  padding: 10px 13px;
+  border-right: 1px solid black;
+}
+
+.action1-4:last-child {
+  margin-right: 0;
+  border-right: 0px;
+}
+
+.welcomemsg {
```

Comment:
Use snake-case for class names. This makes it easier to read quickly and is a understood practice for classes. EX. welcome-msg